src/main/java/org/apache/commons/text/lookup/DefaultStringLookup.java

```diff
@@ -31,100 +34,125 @@
 public enum DefaultStringLookup {
 
     /**
-     * The lookup for Base64 decoding using the key {@value StringLookupFactory#KEY_BASE64_DECODER}.
+     * The lookup for Base64 decoding using the key {@code "base64Decoder"}.
```

Comment:
Wrong fix here and in subsequent comments, instead, use the FQCN, for example, replace:
```
{@value StringLookupFactory#KEY_BASE64_DECODER}
```
with
```
{@value org.apache.commons.text.lookup.StringLookupFactory#KEY_BASE64_DECODER}
```